athlinks-ontology.ttl

```diff
@@ -115,6 +115,22 @@ schema:subjectOf a owl:ObjectProperty ;
   rdfs:comment "A creative work or event about this thing."@en ;
   rdfs:label "subjectOf" .
 
+##
+## step count
+:steps a owl:DatatypeProperty;
+  rdfs:comment "step counts for an individual.";
+  rdfs:label "step counts for an individual.";
+	rdfs:domain [
+    rdf:type owl:Class;
+    owl:unionOf (
+      :AthleticEventCompetitor
```

Comment:
No need to do this. Putting this on person is sufficient given this is a virtual endpoint.

Reply:
I'm going to resolve this myself and merge this to meet time constraints.